elements/lisk-p2p/src/peer/outbound.ts

```diff
@@ -194,6 +194,15 @@ export class OutboundPeer extends Peer {
 		// Bind RPC and remote event handlers
 		outboundSocket.on(REMOTE_SC_EVENT_RPC_REQUEST, this._handleRawRPC);
 		outboundSocket.on(REMOTE_SC_EVENT_MESSAGE, this._handleRawMessage);
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access,@typescript-eslint/no-unsafe-assignment
+		const transportSocket = (outboundSocket as any).transport;
+		// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access
+		if (transportSocket?.socket) {
+			// eslint-disable-next-line @typescript-eslint/no-unsafe-member-access,@typescript-eslint/no-unsafe-call
+			transportSocket.socket.on(REMOTE_EVENT_PING, () => this.applyPenalty(100));
```

Comment:
dont we need to unbound them?

Reply:
`transportSocket.socket` is cleaned up by socketcluster internally so we don't need it